dparc.tex

```diff
@@ -77,10 +77,10 @@ \subsubsection{Satellite Catalog Sub-Zone} \label{sec:satcat}
 
 \subsubsection{ Offline Production Zone}
 
-The Offline Production zone holds data that is not embargoed but is not yet released to the data rights community.
+The Offline Production \gls{Zone} holds data that is not embargoed but is not yet released to the data rights community.
```

Comment:
zone and Zone have been used .. should we settle on Zone ?  I did not change all of them ..